src/utils/command.js

```diff
@@ -127,7 +127,19 @@ class BaseCommand extends Command {
         offline: argv.offline,
       })
     } catch (error) {
-      const message = error.type === 'userError' ? error.message : error.stack
+      const isUserError = error.type === 'userError'
+
+      // If we're failing due to an error thrown by us, it might be because the token we're using is invalid.
+      // To account for that, we try to retrieve the config again, this time without a token, to avoid making
+      // any API calls.
+      //
+      // @todo Replace this with a mechanism for calling `resolveConfig` with more granularity (i.e. having
+      // the option to say that we don't need API data.)
```

Comment:
Would the `offline` flag of `@netlify/config` work here?

Reply:
Yes, that works too. Done in https://github.com/netlify/cli/pull/1810/commits/1c8cc6d1ffec91a27d47199d37022b551d35895d.

Thanks @ehmicky! 👊🏻